docs/beta/tutorial/get_version.md

```diff
@@ -0,0 +1,53 @@
+# Tutorial: Get Tamr version
+This tutorial will introduce basic usage of the Tamr Python client.  It will guide you through the process of configuring the connection to a Tamr instance and retrieving the version of that instance.
+
+## Preparation
+To complete this tutorial you will need access to a Tamr instance.  Specifically you should have a username and password that allow you to log in to Tamr, as well as the socket address of the instance.  The socket address is composed of
+1. The protocol, such as `"https"` or `"http"`
+2. The host, which may be `"localhost"` if the instance is deployed from the same machine from which your Python code will be run
+3. The port at which you access the Tamr user interface, typically `9100`
+
+When you view the Tamr user interface in a browser, the url is `<protocol>://<host>:<port>`.
+
+## Procedure
+### The Session Object
+The Tamr Python client uses a `Session` primarily to persist the user's authentication details across requests made to the server where Tamr is hosted.
+
+A `Session` carries authentication credentials derived from a username and password, and is not explicitly tied to any single Tamr instance. For more details, see the documentation for the [Requests library](https://requests.readthedocs.io/en/master/user/advanced/#session-objects).
+
+ - Use your username and password to create an instance of `tamr_client.UsernamePasswordAuth`.
+ - Use the function `tamr_client.session.from.auth` to create a `Session`.
+```python
+import tamr_client as tc
+import os
+
+username = os.environ['TAMR_USERNAME']
+password = os.environ['TAMR_PASSWORD']
```

Comment:
For a simpler tutorial, it might be better to do something like [getpass](https://docs.python.org/3.8/library/getpass.html) so that users don't need to understand environment variable management in Python.

Reply:
I agree with @pcattori. In the past I created these environment variables before creating a virtual environment, then realized I could not access them. I see that the Toolbox uses [`yaml`](http://10.20.0.250:8000/tamr-toolbox/latest/) for loading these variables.

Reply:
@minroh I think the Toolbox method of using `yaml` is very nice, but maybe more than we want to include in this tutorial.

@pcattori `input` seems pretty straightforward, but I'm slightly hesitant to have the user import another module like `getpass`.  Is the idea that this portion of the code would be along the lines of the following, where the execution involves responding to command-line prompts during runtime?
```python
import getpass

username = input("Username:")
password = getpass.getpass("Password:")
```

Reply:
`getpass` is part of the standard library as is `os`, so `getpass` is no worse than `os.environ`.